### PR TITLE
Removing non-Custodian reports and cleaned up removing :report:

### PR DESCRIPTION
**Pull request by LW001**
Quick and easy change :^)

### CODE REVIEW

File: Commands/Commands/queue.js [hunk 1]
```diff
@@ -268,22 +268,21 @@ commands.registerVote = {
...
                       name: 'reported',
                       id: '323058409203171328'
                     })
                     doc.type = 'adminReviewDelete'
                     switchIDs(doc, bot)
+                  } else {
+                    msg.removeReaction({
+                      id: '302137374920671233',
+                      name: 'report'
+                    }, user)
```
Comment (SteamingMutt) [inline Commands/Commands/queue.js]: What's user?
Is the intended behaviour here to only remove 1 reaction?

File: Commands/Commands/queue.js [hunk 2]
```diff
@@ -268,22 +268,21 @@ commands.registerVote = {
                     result: (doc.reports === config.discord.reportThreshold) ? 'Report has been sent to admins' : undefined
                   })
                   if (doc.reports === config.discord.reportThreshold) {
-                    var reportsArr = msg.fetchReactions({
+                    msg.removeReaction({
```
Comment (Dougley) [inline Commands/Commands/queue.js]: Remove own reaction only?

**Comment by RSNFreud** [comment]
Whats this waiting for?

File: Commands/Commands/queue.js [hunk 3]
```diff
@@ -271,7 +271,7 @@ commands.registerVote = {
                     var reportsArr = msg.fetchReactions({
                       id: '302137374920671233',
                       name: 'report'
-                    }, bot.User)
```
Comment (Dougley) [inline Commands/Commands/queue.js:274]: `bot.User` would be the correct usage, refer to the [discordie documentation.](qeled.github.io/discordie/)

**Review by Dougley — CHANGES_REQUESTED** [review]

**Comment by Dougley** [comment]
Also, please resolve the merge conflicts.

**Comment by Dougley** [comment]
No activity for nearly a month, closing.

**Pull request closed without merging**